### ISSUE TITLE
Audio is distorted

### ISSUE THREAD

**Issue by ROCKNROLLKID** [GardensOfKadesh/Homeworld#21]
When I launch the game through browser, the audio is really distorted. I am playing through Microsoft Edge browser on Windows 11.

**Comment by ThibaultLemaire** [comment]
Distorted how?

Choppy sound on slow processors is a known issue (though thank you for opening a ticket here, we have only been discussing this on the Discord IIRC). It has to do with the single-threadedness of web assembly. At this point the best I can offer you is to try to compile a native build

**Comment by ROCKNROLLKID** [comment]
I made a short video to show you what I mean: https://www.youtube.com/watch?v=f9bhr_9tGf4

I have a Ryzen 5800x and I also have a Sound Blaster Audigy RX if that helps.

**Comment by ThibaultLemaire** [comment]
Thanks for the vid, very much appreciated.

That is not what I had in mind, and definitely something I haven't seen before (or heard rather). With that hardware, performance shouldn't be the issue either. (I don't believe the sound card matters)

Have you tried with other browsers?

**Comment by ROCKNROLLKID** [comment]
I just installed google chrome with no extensions or anything, google chrome has the same problem.

I also just installed Firefox to see, weirdly Firefox does not have this problem. Audio is totally normally on Firefox.

So maybe some issue with chrome based browsers?

EDIT: I just tested some other web assembly games and so far I have not noticed any audio distortion with any other game. So far this is the only game.
